pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -92057,7 +92057,7 @@ self: {
      }) {inherit (pkgs) gtk3;};
 
   "gi-gdkpixbuf" = callPackage
-    ({ mkDerivation, base, bytestring, Cabal, containers, gdk_pixbuf
+    ({ mkDerivation, base, bytestring, Cabal, containers, gdk-pixbuf
```

Comment:
You'll have to change this in `hackage2nix` here: https://github.com/NixOS/cabal2nix/blob/f4355aa45723abd15b29eb826800839278f91ed9/src/Distribution/Nixpkgs/Haskell/FromCabal/Name.hs#L50

Fixing it in this file will just make it reappear the next time it's regenerated.

Reply:
Opened https://github.com/NixOS/cabal2nix/pull/424